src/manifolds/FixedRankMatrices.jl

```diff
@@ -408,3 +407,21 @@ function zero_tangent_vector!(
     X.Vt .= zeros(eltype(X.Vt), k, n)
     return X
 end
+
+@doc raw"""
+    to_ambient(X::SVDMPoint)
+
+Reconstruct a low-rank matrix from the its `SVDMPoint` representation.
+"""
+function to_ambient(X::SVDMPoint)
+    return X.U * diagm(X.S) * X.Vt
```

Comment:
This avoids an allocation
```suggestion
    return X.U * Diagonal(X.S) * X.Vt
```